src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfoCacheService.cs

```diff
@@ -38,40 +48,35 @@ public SymbolTreeInfoCacheService()
             if (metadataId == null)
                 return null;
 
-            var checksum = SymbolTreeInfo.GetMetadataChecksum(solution, reference, cancellationToken);
-
-            // See if the last value produced matches what the caller is asking for.  If so, return that.
-            if (_metadataIdToInfo.TryGetValue(metadataId, out var metadataInfo) &&
-                metadataInfo.SymbolTreeInfo.Checksum == checksum)
-            {
+            // See if the last value produced exactly matches what the caller is asking for.  If so, return that.
```

Comment:
This comment is wrong now isn't it? It just checks if we have any cached value

Reply:
indeedly.